src/tests/utils/mock_data_factory.py

```
@@ -0,0 +1,390 @@
+    ]
+
+    # Sample stances for evidence
+    STANCES = ["supporting", "contradicting", "neutral"]
+
+    # Sample verdict types
+    VERDICT_TYPES = ["true", "false", "partially true", "unverifiable"]
+
+    @classmethod
+    def create_claim(
+        cls,
+        text: Optional[str] = None,
```

Ruff (0.11.9)

Use `X | Y` for type annotations

Convert to `X | Y`

(UP007)